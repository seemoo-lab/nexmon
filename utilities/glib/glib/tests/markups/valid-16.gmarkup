<a>
  <b>
    <c1>c1</c1>
    <c2>c2</c2>
  </b>
  <b1>b1</b1>
  <b2>b2</b2>
  <b3>b3</b3>
  <b4><![CDATA[This is CDATA]]></b4>
</a>
