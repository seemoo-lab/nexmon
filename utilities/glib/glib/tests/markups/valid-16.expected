ELEMENT 'a'
  TEXT '
  '
  ELEMENT 'b'
    TEXT '
    '
    ELEMENT 'c1'
      TEXT 'c1'
    END 'c1'
    TEXT '
    '
    ELEMENT 'c2'
      TEXT 'c2'
    END 'c2'
    TEXT '
  '
  END 'b'
  TEXT '
  '
  ELEMENT 'b1'
    TEXT 'b1'
  END 'b1'
  TEXT '
  '
  ELEMENT 'b2'
    TEXT 'b2'
  END 'b2'
  TEXT '
  '
  ELEMENT 'b3'
    TEXT 'b3'
  END 'b3'
  TEXT '
  '
  ELEMENT 'b4'
    TEXT ''
    PASS '<![CDATA[This is CDATA]]>'
    TEXT ''
  END 'b4'
  TEXT '
'
END 'a'
